rome-modules/src/test/resources/itunes/leshow.xml

```diff
@@ -65,7 +66,11 @@
 	
 	    <itunes:duration>46:34</itunes:duration>
 	
-    <itunes:keywords></itunes:keywords>  
+    <itunes:keywords></itunes:keywords>
+    <itunes:encoded><![CDATA[show notes with <b>bold</b> text.]]></itunes:encoded>
```

Comment:
The pdf says it's `<content:encoded>`. I don't know what they mean by the `content` namespace, but isn't it suspicious that it's different from all other tags?

Reply:
Looks like we already have this tag in the `content` module: https://github.com/rometools/rome/blob/master/rome-modules/src/main/java/com/rometools/modules/content/ContentModuleImpl.java#L43